README.md

```diff
@@ -2,192 +2,128 @@
-- login to azure docker registry either via admin user
-    ```sh
-    docker login omnectweucopsacr.azurecr.io
-    ```
-    or via your AAD account
-    ```sh
-    az login
-    az acr login -n omnectweucopsacr
-    ```
-- Pull latest docker image
-    ```sh
-    docker pull omnectweucopsacr.azurecr.io/omnect-cli-backend:latest
-    ```
-If you want to use a specific version, look for available versions in the [registry](https://portal.azure.com/#@conplementag2.onmicrosoft.com/resource/subscriptions/ebaba6ec-e467-4409-b73e-bdd5e34a34c1/resourceGroups/omnect-weu-cops-acrrg/providers/Microsoft.ContainerRegistry/registries/omnectweucopsacr/overview).
+- Device Update for IoT Hub configuration: inject [`du-config.json`](https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-configuration-file)
+- Generic configuration of services
+  - copy files to image in order to configure e.g. boot service, firewall, wifi and others
+  - copy files from image, e.g. to patch and re-inject configurations
+- SSH: 
+  - inject a ssh root ca and device principal for ssh tunnel creation
+  - open a ssh tunnel on a device in the field to connect to it
 
 # Installation
-Ensure ~/bin/ exists and is in your $PATH before executing:
 
+Available debian packages can be listed as a xml document via this [link](https://omnectassetst.blob.core.windows.net/omnect-cli?restype=container&comp=list). Choose, download and install a version:
 ```sh
-docker run --rm --entrypoint cat omnectweucopsacr.azurecr.io/omnect-cli-backend:latest /install/omnect-cli > ~/bin/omnect-cli && chmod +x ~/bin/omnect-cli
+wget https://omnectassetst.blob.core.windows.net/omnect-cli/omnect-cli_<version>_amd64.deb
+sudo dpkg -i omnect-cli_<version>_amd64.deb
 ```
+**Note**: `dpkg` lists necessary runtime dependencies in case they are not present.
 
-# Identity configuration
-## Configuration of devices NOT part of a gateway with leaf scenario
-For `omnect-iot-devices` and `omnect-iotedge-devices` adapt [config.toml.est.template](conf/config.toml.est.template) or [config.toml.tpm.template](conf/config.toml.tpm.template) to your needs.
-
-```sh
-omnect-cli identity set-config -c <path>/config.toml -i <path>/image.wic
-
-Options:
-  -p <path>/payload.json
-  -b create bmap file
-```
-For further information on using dps payloads read the following [link](https://learn.microsoft.com/de-de/azure/iot-dps/concepts-custom-allocation).
+# Build from sources
 
-## Prepare `omnect-iotedge-gateway-device` and `omnect-iot-leaf-device` for a transparent gateway with leaf scenario
-Follow this article [Configure an IoT Edge device to act as a transparent gateway](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-create-transparent-gateway?view=iotedge-2020-11) to understand the iotedge based transparent gateway setup. We assume that you use a X.509 CA certificate setup.
+The application can be built via `cargo` as usual. As a prerequisite `libmagic-dev` package must be installed on host system.
```

Comment:
if you build from sources, the dependency is libmagic ... 
libmagic-dev is a special package of debian based (host) buildsystems.

Reply:
"A prerequisite is `libmagic`, e.g. the package `libmagic-dev` must be installed on a debian-based host system."
?